pom-tuner/src/test/java/org/l2x6/pom/tuner/SomeSimpleTest.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2015 Maven Utilities Project
+ * project contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.l2x6.pom.tuner;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class SomeSimpleTest {
+
+    @Test
+    void smoke() {
+        assertTrue(true);
```

Comment:
What is the purpose of this test?

Reply:
Nothing, just wanted to add one single UT here... but the point is:
* pom-tuner UTs (like this SomeSimpleTest) would be executed ONLY in "install" job (so only on one OS, one Java).
* the "integration test-like" tests are moved to pom-tuner-tests and those are executed on full matrix.